addons/block_code/drag_manager/drag.gd

```diff
@@ -3,6 +3,7 @@ extends Control
 
 const Background = preload("res://addons/block_code/ui/blocks/utilities/background/background.gd")
 const BlockCanvas = preload("res://addons/block_code/ui/block_canvas/block_canvas.gd")
+const BlockTreeUtil = preload("res://addons/block_code/ui/block_tree_util.gd")
```

Comment:
I'm not at all convinced of this name or path. What seemed to be in common in the functions moved was that they were about a tree of `Block` nodes. @wnbaum's work completely changes the way this is handled.

Reply:
If this is going to be changed anyways maybe the current name/path doesn't matter that much?